packages/autorest.gotest/src/generator/exampleGenerator.ts

```diff
@@ -8,10 +8,36 @@ import { Config } from '../common/constant';
 import { ExampleModel, MockTestDefinitionModel } from '@autorest/testmodeler/dist/src/core/model';
 import { MockTestDataRender } from './mockTestGenerator';
 import { ParameterOutput } from '../common/model';
-import { camelCase } from 'lodash';
+import { camelCase, values } from 'lodash';
 import _ = require('lodash');
+import { ObjectSchema, Parameter, SchemaType } from '@autorest/codemodel';
+import { sortParametersByRequired } from '@autorest/go/dist/generator/helpers';
```

Comment:
Could you use prettier to format the code?

Reply:
I am not sure about the current style

Reply:
Actually, the code has already been formatted. When migrating to this repo, I follow the exists lint rule and it seems quite loosen.